grpc/setup.py

```diff
@@ -120,7 +120,7 @@ def make_packages(root_dir):
     license='Apache License 2.0',
     python_requires='>=3.7',
     install_requires=[
-        'th2-grpc-common==4.3.0.dev0',
+        'th2-grpc-common==4.5.0.dev0',
```

Comment:
issue(blocking): Incorrect version
```suggestion
        'th2-grpc-common==4.5.0rc1',
```